app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+# from app.errors import * bad practice or something?
+from app.errors import NotVaccinatedError, OutdatedVaccineError
```

Comment:
You do not need to import both of these classes, because they are subclasses of the `VaccineError` superclass. You can write: `from app.errors import VaccineError, NotWearingMaskError` and delete the next line

Reply:
Yes thank you!